src/braket/experimental/autoqasm/program/gate_calibrations.py

```diff
@@ -0,0 +1,86 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Callable, Tuple
+
+import braket
+from braket.experimental.autoqasm import program as aq_program
+from braket.pulse import PulseSequence
+
+if TYPE_CHECKING:
+    from braket.experimental.autoqasm import Qubit
+
+
+class GateCalibrations:
+    # TODO: Possibly inherits from BDK GateCalibrations
+    """
+    A set of gate calibrations.
+    """
+
+    def __init__(self):
+        self._calibrations = {}
+
+    def register(self, gate_name: str, qubits: Tuple[Qubit], angles: Tuple[float] = ()):
```

Comment:
It would be cool if `qubits` and `angles` also accepted a single `Qubit` or `float` arg instead of requiring the user to pass a tuple. Also it may be better to accept `Iterable` rather than `Tuple`. So something like:
```
qubits: Union[Qubit, Iterable[Qubit]], angles: Union[float, Iterable[float]]
```